config.ini

```diff
@@ -0,0 +1,19 @@
+[Default]
+PORT: 8084
+HOST: 127.0.0.1
+START_CMD: python server.py
+PACKAGES_INSTALL: poppler-utils dejavu-serif-fonts dejavu-sans-mono-fonts dejavu-sans-fonts
+TMP_DIR: tmp
+
+[logging]
+LOGGING_FORMAT: %(asctime)s|%(levelname)s: sdx-transform-cs: %(message)s
+LOGGING_LEVEL: DEBUG
+
+[Service URL]
+SDX_SEQUENCE_URL: http://sdx-sequence:5000
+
+[FTP setup]
+FTP_HOST: \\\\NP3RVWAPXX370\\SDX_preprod\\
```

Comment:
This doesn't seem right? It's a path not a host? 

(probably shouldn't be in open configuration either)

Reply:
This is the default in the settings file

Reply:
I felt the same when I first saw it, but it's been in the default settings file the whole time.  I think it might be a good idea to remove it

Reply:
It's already elsewhere in the code so lets just leave it for now and we'll remove it later